sum_of_multiples.rb

```diff
@@ -0,0 +1,27 @@
+class SumOfMultiples
+
+  def initialize(*num)
+    @array_of_numbers = num
+  end
+
+  def to(multiple_limit)
+    @result_array = []
+    sum_of_multiples=0
+    (0..@array_of_numbers.length-1).each do |i|
+      if multiple_limit > @array_of_numbers[i]
+        j = 1
+        while (@array_of_numbers[i] * j) < multiple_limit
+          a = @array_of_numbers[i] * j
+          @result_array.push(a) if !@result_array.include?(a)
+          j += 1
+        end
+      else
```

Comment:
@HardikPurohit No need of else condition. You have already defined the local variable outside the `if` condition.